contracts/mocks/ERC20Mock.sol

```diff
@@ -1,33 +1,16 @@
 // SPDX-License-Identifier: MIT
-
 pragma solidity ^0.8.0;
 
-import "../token/ERC20/ERC20.sol";
+import {ERC20} from "../token/ERC20/ERC20.sol";
```

Comment:
I believe that is the first time we use this syntax. Is that something we want to be doing more ?

Reply:
Not sure. It wasn't a conscious decision, I think I went along with the syntax from `a16z/erc4626-tests`.